.travis.yml

```diff
@@ -18,9 +18,7 @@ addons:
     token:
       secure: $SONAR_SECURITY_TOKEN
     branches:
-      - master
-      - develop
-      - refactor
+      - /.*/ 
```

Comment:
I changed it so that it build all branches regardless. All other review comments can be addressed in the next sprint as far as I'm concerned if we do not have time to address it before the end of this sprint.